spec/models/book_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  it do
+    should validate_presence_of(:genre)
+  end
+  it do
+    should validate_presence_of(:author)
+  end
+  it do
+    should validate_presence_of(:image)
+  end
+  it do
+    should validate_presence_of(:title)
+  end
+  it do
+    should validate_presence_of(:publisher)
+  end
+  it do
+    should validate_presence_of(:year)
+  end
+
+  subject(:book) do
+    described_class.new(
+      genre: genre,
+      author: author,
+      image: image,
+      title: title,
+      publisher: publisher,
+      year: year
+    )
+  end
+
+  let(:genre)     { Faker::Book.genre }
+  let(:author)    { Faker::Book.author }
+  let(:image)     { Faker::Internet.url('image-host.com', '/image.png') }
+  let(:title)     { Faker::Book.title }
+  let(:publisher) { Faker::Book.publisher }
+  let(:year)      { Faker::Number.between(1800, 2017) }
+
+  it 'is valid with valid parameters' do
+    is_expected.to be_valid
+  end
+
+  describe '#create' do
```

Comment:
None of this test are required, you only test what you code. We can assume `:create` is already testes by Rails tests. Checking validations it's enough in a newly created model.